openedx_tagging/core/tagging/api.py

```diff
@@ -107,3 +106,68 @@ def tag_object(
     """
 
     return taxonomy.tag_object(tags, object_id, object_type)
+
+
+def autocomplete_tags(taxonomy: Taxonomy, prefix: str, object_id: str= None, count=10) -> QuerySet:
```

Comment:
@ChrisChV  Can we move the bulk of this functionality into the Taxonomy model? That way, if system taxonomies (for example) need to customize anything, they can just overwrite the method.

Reply:
Updated here https://github.com/openedx/openedx-learning/pull/59/commits/223b3a4c314d84255a36e73cd80d9b108e577568